lib/genres.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'movie.dart';
+
+class Genres extends StatelessWidget {
+  final Movie _mov;
+
+  const Genres(this._mov, {super.key});
+
+  static const genresWordPadding = EdgeInsets.only(left: 20);
+  static const genresTextStyle = TextStyle(
+    color: Colors.white,
+    fontWeight: FontWeight.bold,
+    fontSize: 22,
+  );
```

Comment:
if you have more styles like this, you can move them into a styles files, like the constants

Reply:
Did it, created the new file with the TextStyles